src/Library/Contracts/opencensus/code/Common.g.cs

```diff
@@ -24,29 +24,29 @@ public static partial class CommonReflection {
     static CommonReflection() {
       byte[] descriptorData = global::System.Convert.FromBase64String(
           string.Concat(
-            "Ci1vcGVuY2Vuc3VzL3Byb3RvL2FnZW50L2NvbW1vbi92MS9jb21tb24ucHJv",
-            "dG8SIG9wZW5jZW5zdXMucHJvdG8uYWdlbnQuY29tbW9uLnYxGh9nb29nbGUv",
-            "cHJvdG9idWYvdGltZXN0YW1wLnByb3RvItgCCgROb2RlEkcKCmlkZW50aWZp",
-            "ZXIYASABKAsyMy5vcGVuY2Vuc3VzLnByb3RvLmFnZW50LmNvbW1vbi52MS5Q",
-            "cm9jZXNzSWRlbnRpZmllchJDCgxsaWJyYXJ5X2luZm8YAiABKAsyLS5vcGVu",
-            "Y2Vuc3VzLnByb3RvLmFnZW50LmNvbW1vbi52MS5MaWJyYXJ5SW5mbxJDCgxz",
-            "ZXJ2aWNlX2luZm8YAyABKAsyLS5vcGVuY2Vuc3VzLnByb3RvLmFnZW50LmNv",
-            "bW1vbi52MS5TZXJ2aWNlSW5mbxJKCgphdHRyaWJ1dGVzGAQgAygLMjYub3Bl",
-            "bmNlbnN1cy5wcm90by5hZ2VudC5jb21tb24udjEuTm9kZS5BdHRyaWJ1dGVz",
-            "RW50cnkaMQoPQXR0cmlidXRlc0VudHJ5EgsKA2tleRgBIAEoCRINCgV2YWx1",
-            "ZRgCIAEoCToCOAEiaAoRUHJvY2Vzc0lkZW50aWZpZXISEQoJaG9zdF9uYW1l",
-            "GAEgASgJEgsKA3BpZBgCIAEoDRIzCg9zdGFydF90aW1lc3RhbXAYAyABKAsy",
-            "Gi5nb29nbGUucHJvdG9idWYuVGltZXN0YW1wIpsCCgtMaWJyYXJ5SW5mbxJI",
-            "CghsYW5ndWFnZRgBIAEoDjI2Lm9wZW5jZW5zdXMucHJvdG8uYWdlbnQuY29t",
-            "bW9uLnYxLkxpYnJhcnlJbmZvLkxhbmd1YWdlEhgKEGV4cG9ydGVyX3ZlcnNp",
-            "b24YAiABKAkSHAoUY29yZV9saWJyYXJ5X3ZlcnNpb24YAyABKAkiiQEKCExh",
-            "bmd1YWdlEhgKFExBTkdVQUdFX1VOU1BFQ0lGSUVEEAASBwoDQ1BQEAESCwoH",
-            "Q19TSEFSUBACEgoKBkVSTEFORxADEgsKB0dPX0xBTkcQBBIICgRKQVZBEAUS",
-            "CwoHTk9ERV9KUxAGEgcKA1BIUBAHEgoKBlBZVEhPThAIEggKBFJVQlkQCSIb",
-            "CgtTZXJ2aWNlSW5mbxIMCgRuYW1lGAEgASgJQn8KI2lvLm9wZW5jZW5zdXMu",
-            "cHJvdG8uYWdlbnQuY29tbW9uLnYxQgtDb21tb25Qcm90b1ABWklnaXRodWIu",
-            "Y29tL2NlbnN1cy1pbnN0cnVtZW50YXRpb24vb3BlbmNlbnN1cy1wcm90by9n",
-            "ZW4tZ28vYWdlbnQvY29tbW9uL3YxYgZwcm90bzM="));
+            "ChxhZ2VudC9jb21tb24vdjEvY29tbW9uLnByb3RvEiBvcGVuY2Vuc3VzLnBy",
```

Comment:
i wonder can we use this string as a unique identifier of the proto version instead of the commit hash?

Reply:
Can we find it somewhere in the original repo with protos?